src/fields/Field.cpp

```diff
@@ -1,13 +1,9 @@
 #include "Field.h"
 
-Field::Field(string fieldName, FieldType fieldType, bool isPrimaryKey, bool isUnique, bool isNullable, bool isAutoFilled) {
-    _fieldName = fieldName;
-    _fieldType = fieldType;
-    _isPrimaryKey = isPrimaryKey;
-    _isUnique = isUnique;
-    _isNullable = isNullable;
-    _isAutoFilled = isAutoFilled;
-}
+Field::Field(string &fieldName, FieldType fieldType, bool isPrimaryKey, bool isUnique, bool isNullable, bool isAutoFilled) :
+    _fieldName(fieldName), _fieldType(fieldType), _isPrimaryKey(isPrimaryKey), _isUnique(isUnique), _isNullable(isNullable), _isAutoFilled(isAutoFilled) {
+        _isNull = true;

```

Comment:
It can, but g++ gives a warning:
```
warning: ‘Field::_isAutoFilled’ will be initialized after [-Wreorder]
         bool _isAutoFilled;
              ^
../src/fields/Field.h:22:14: warning:   ‘bool Field::_isNull’ [-Wreorder]
         bool _isNull;
```
Apart from this it does run fine.